### DISCUSSION TITLE
Change tabs on hover (Nested Tabs)

### DISCUSSION THREAD

**Discussion by 79ho3ein** [elementor/elementor#22728] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Currently we can switch between tabs only by clicking. It would be great if we could change tabs on hover.

### Describe the solution you'd like

Change tabs on hover in Nested Tabs widget.

**Comment by swmaguirpi** [comment]
Second this as a very useful feature that I would use. I suspect there is a way to do this with custom code but I have not yet figured it out.
